.github/workflows/test-integration.yml

```diff
@@ -34,16 +34,25 @@ jobs:
     - name: Build worker-controller image
       run: |
         skaffold build --profile worker-controller --push=false
+      env:
+        DOCKER_BUILDKIT: 1
```

Comment:
it seems like somehow this isn't being passed to the skaffold build job, which is failing with `"docker build failure: the --mount option requires BuildKit. Refer to https://docs.docker.com/go/buildkit/ to learn how to build images with BuildKit enabled. Please fix the Dockerfile and try again.."`